pkg/master/master.go

```diff
@@ -781,11 +782,14 @@ func (m *Master) api_v1() *apiserver.APIGroupVersion {
 func (m *Master) expapi(c *Config) *apiserver.APIGroupVersion {
 	controllerStorage := expcontrolleretcd.NewStorage(c.DatabaseStorage)
 	autoscalerStorage := horizontalpodautoscaleretcd.NewREST(c.DatabaseStorage)
+	nodeMetricsStorage := nodemetrics.NewREST(c.DatabaseStorage)
 
 	storage := map[string]rest.Storage{
 		strings.ToLower("replicationControllers"):       controllerStorage.ReplicationController,
 		strings.ToLower("replicationControllers/scale"): controllerStorage.Scale,
 		strings.ToLower("horizontalpodautoscalers"):     autoscalerStorage,
+		// /api/experimental/nodeMetrics/{myNode}
```

Comment:
Since github doesn't show that comment properly since it's hidden, I'll just copy the comment.

@nikhiljindal said "Not sure if we enforce it, but its lowercase for all our existing resources."